infra/modules/service/events_role.tf

```diff
@@ -1,8 +1,7 @@
-#---------------------
-# Task Scheduler Role
-#---------------------
+#------------
+# Events Role
```

Comment:
refactors step function auth in a such a way that risks them failing

Reply:
Same as https://github.com/HHS/simpler-grants-gov/pull/3925#discussion_r1987899941